index.html

```diff
@@ -184,14 +184,15 @@ <h2 class="section-title title-l1">About me</h2>
   </section>
   <footer>
     <div id="contact" class="page-section contact-section">
-      <form class="contact-form" action="https://formspree.io/f/mzbopkyk" method="post">
+      <form id="contact-form" class="contact-form" action="https://formspree.io/f/mzbopkyk" method="post">
```

Comment:
- [ ] Nice job here. However, following the requirement: The form `"submit"` or the submit button/input `"click"` event is used to attach the function checking the email format. We need to manually create the submit function in our javascript file and then use it here. Check this [page](https://www.w3schools.com/jsref/event_onsubmit.asp) for more information.

- [ ] When you implement the submit function in your javascript. I will suggest you also account for this feature:```The students can check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string. ```

Reply:
Thanks for your feedback.
I checked out the link you shared. I am using the **object.addEventListener("submit", myScript);** method to validate the form's controls in javascript.
I don't understand well what you mean.

Reply:
Hi Gabyse1,
I don't mean `validation`. Here we want to submit the form after filling the form. So you should create a function that submits the form Instead of using this external link:` action="https://formspree.io/f/mzbopkyk"`. Let me know if you need further clarifications.

Reply:
Hi Enaburekhan,
I kindly ask you to give me further clarifications because I need to use the formspree.io service to handle the information and send it to my email. If I don't use this service, where do I submit the form?. Or do you mean to include the action in the event listener?
Thank you again.

Reply:
Ok Gabyse1, I never knew the `formspree.io` was sending the information to your email. Sorry about that. However, my suggestion is this, wrap this 
```
 if (emailv.value.trim().match(/[A-Z]/g)) {
    errorSpan.textContent = 'Email field should not have capital letters.';
    errorSpan.classList.add('visible');
    emailv.classList.add('field-error');
  } else {
    contactForm.submit();
  }
});
``` 
```
in a function. then add this to the form like this 
<form onsubmit= function name  --everything stays the same-->
  Contents remains the same
</form>
```
Hope this helps

Reply:
I understand your suggestion, but I've already done the same when I call the **addEventlistener('submit', () =>...)**, because I'm catching the same action. You can notice that the preventDefault inside it doesn't allow to send the form until everything email meets the requirements. In the page you shared, it also suggests that way to use as an alternative to **onsubmit** in HTML.